src/my_package/CMakeLists.txt

```diff
@@ -0,0 +1,199 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(my_package)
```

Comment:
Looks like an example package slipped into the commit?

Reply:
Deleted